src/acb_poly/powsum_series_naive_threaded.c

```diff
@@ -120,24 +122,24 @@ _acb_zeta_powsum_evaluator(void * arg_ptr)
 
     flint_cleanup();
 
-    return NULL;
+    return;
 }
 
 void
 _acb_poly_powsum_series_naive_threaded(acb_ptr z,
     const acb_t s, const acb_t a, const acb_t q, slong n, slong len, slong prec)
 {
-    pthread_t * threads;
-    powsum_arg_t * args;
-    slong i, num_threads;
+    thread_pool_handle * handles;
+    _worker_arg * args;
+    slong i, num_threads, num_workers;
     int split_each_term;
 
-    num_threads = flint_get_num_threads();
+    split_each_term = (len > 1000);
 
-    threads = flint_malloc(sizeof(pthread_t) * num_threads);
-    args = flint_malloc(sizeof(powsum_arg_t) * num_threads);
+    num_workers = flint_request_threads(&handles, split_each_term ? len - 1 : n);
```

Comment:
Is this supposed to be ``len - 1`` or ``len``?

Reply:
It is supposed to be `len - 1`, but it is also supposed to be `n - 1`. Thanks!